packages/split-layout/test/split-layout.common.js

```diff
@@ -73,6 +73,96 @@ describe('split layout', () => {
       expect(getComputedStyle(first).pointerEvents).to.equal('visible');
       expect(getComputedStyle(second).pointerEvents).to.equal('visible');
     });
+
+    describe('elements with slot pre-defined', () => {
+      it('should respect pre-defined slot values in both elements', async () => {
+        const layout = fixtureSync(`
+          <vaadin-split-layout>
+            <div id="second" slot="secondary">secondary</div>
+            <div id="first" slot="primary">primary</div>
+          </vaadin-split-layout>
+        `);
+        await nextRender();
+        expect(layout.querySelector('#first').getAttribute('slot')).to.be.equal('primary');
```

Comment:
**nit:** `.getAttribute('slot')` could be shortened to `.slot`
```suggestion
        expect(layout.querySelector('#first').slot).to.be.equal('primary');
```

Reply:
Changed to use the `slot` property in all places.